llama_stack/models/llama/llama4/chat_format.py

```diff
@@ -216,12 +210,9 @@ def _process_content(c):
                 content = ToolUtils.encode_tool_call(t, tool_prompt_format)
                 _process_content(content)
 
-        # Tool calls and Tool Response messages should be eom
```

Comment:
@hardikjshah this got reverted because the change you made for eom / eot should be made in llama-models also. I will make sure that happens, but this kind of thing is exactly why I want to make this PR. It should be obvious when we are changing llama-models sourced things (and can have automation to detect later)